### DISCUSSION TITLE
请问fork源码后如何编译 需要什么环境

### DISCUSSION THREAD

**Discussion by AlexFugui** [SoftEtherVPN/SoftEtherVPN#1814] [General]
问题和使用无关
只是想自己编译一个版本
对一些参数做固定而不是可以修改的
做java开发的有一定代码基础
只是不知道编译这个需要什么工具和环境
有了解的朋友烦请告知谢谢

**Comment by chipitsine** [comment]
not a bug.
let it reside in "discussion"

**Comment by quantrpeter** [comment]
Do you think they know chinese language?

**Reply by shipitsin-ilia** [reply]
chances are that chinese community can help.
